File: onlineTest-2/src/main/java/com/capg/onlinetest/service/QuestionService.java
package com.capg.onlinetest.service;

import java.util.List;
import com.capg.onlinetest.entity.Question;

/**
 * 
 * @author TEAM EINSTEIN
 * QUESTION SERVICE INTERFACE
 */
public interface QuestionService {

	public String addQuestion(int testId,Question question);
    public String deleteQuestion(int questionId);
    public String updateQuestion(int questionId,Question question);
    public List<Question> viewAllQuestions();
}

File: OnlineTestMgtSystem/src/app/service/question.service.ts
import { Injectable } from '@angular/core';
import { HttpClient } from '@angular/common/http'

@Injectable({
  providedIn: 'root'
})
export class QuestionService {

  private baseUrl = 'http://localhost:9090/test'

  constructor(private http:HttpClient) { }
public  getQuestions(){
  return this.http.get("http://localhost:9090/admin/viewAllQuestions");
}
public deleteQuestion(questionId){
  return this.http.delete("http://localhost:9090/admin/deleteQuestion/"+questionId);
}
public addQuestion(testId, question){
  return this.http.post(`http://localhost:9090/admin/addQuestion/${testId}`,question,{responseType:'text' as 'json'});
}
}